main.tf

```diff
@@ -1,14 +1,23 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 data "google_project" "project" {
   project_id     = var.project_id
 }
 
-terraform {
-  backend "gcs" {
-    bucket  = null
-    prefix  = null
-  }
-}
-
 resource "google_project_service" "serviceusage" {
```

Comment:
nit: we can use project services to express this succinctly

https://github.com/terraform-google-modules/terraform-google-project-factory/tree/master/modules/project_services